plugins/win-capture/graphics-hook/gl-capture.c

```diff
@@ -827,6 +836,9 @@ static BOOL WINAPI hook_wgl_swap_layer_buffers(HDC hdc, UINT planes)
 
 static BOOL WINAPI hook_wgl_delete_context(HGLRC hrc)
 {
+	if (!gl_functions_initialized)
+		return FALSE;
```

Comment:
Style:  For consistency with other `return` statements, write "FALSE" in lowercase letters.